src/components/form.js

```diff
@@ -1,14 +1,23 @@
+import React from 'react';
 import './form.css';
+import { useDispatch } from 'react-redux';
+import { addBook, Book } from '../redux/books/books';
 
-const Form = () => (
-  <form className="add-form">
-    <h2 className="add-Title">ADD NEW BOOK</h2>
-    <div className="input-area">
-      <input type="text" placeholder="Book title" />
-      <input type="text" placeholder="Category" />
-      <button type="submit"><span className="ADD-BOOK">ADD BOOK</span></button>
-    </div>
-  </form>
-);
+const Form = () => {
+  const dispatch = useDispatch();
```

Comment:
- [ ] The Input form should have a local state for `title` and `author`, and the local state should be updated every time the input changes using `onChange`. e.g. 
``` 
const [title, setTitle] = useState('');
const titleChange = (e) => setTitle(e.target.value);

<input type="text" onChange={titleChange} placeholder="Book title" />
```